proto/vtctldata.proto

```diff
@@ -701,6 +712,45 @@ message ReparentTabletResponse {
   topodata.TabletAlias primary = 3;
 }
 
+message SetShardIsPrimaryServingRequest {
+  string keyspace = 1;
+  string shard = 2;
+  bool is_serving = 3;
+}
+
+message SetShardIsPrimaryServingResponse {
+  // Shard is the updated shard record.
+  topodata.Shard shard = 1;
+}
+
+message SetShardTabletControlRequest {
+  string keyspace = 1;
+  string shard = 2;
+  topodata.TabletType tablet_type = 3;
+  repeated string cells = 4;
+  // DeniedTables updates the list of denied tables the shard will serve for
+  // the given tablet type. This is useful to fix tables that are being blocked
+  // after a vertical split.
```

Comment:
I think it's ok that we still call it vertical split here (as a generic term) versus `MoveTables` which is more of a vitess-specific term.
If you want to change it to `MoveTables` I won't object :)